src/App/App.js

```diff
@@ -28,6 +29,36 @@ class App extends Component {
 			})
 	}
 
+	componentDidUpdate(prevProps, prevState) {
+		if (this.state.isLoggedIn !== prevState.isLoggedIn) {
+			getUserRatings(this.state.id)
```

Comment:
Was the OR statement here really breaking something? we'll have to try it on another machine...(no offense)

Reply:
Hopefully your solid internet connection will be better at finding and killing this BUG. I don't know if the OR statement broke anything? Let's try to add it back in!